packages/core/src/RenderPipeline/MaskManager.ts

```diff
@@ -54,8 +54,10 @@ export class MaskManager {
         }
 
         if (influenceLayers & curMaskLayer) {
+          mask.getMaterial().renderState.renderQueueType = incrementMaskQueue.renderQueueType;
```

Comment:
In `SpriteMask`, `renderQueueFlags` has been set to All.

There may be a better solution?

Reply:
Done